core/lib/config/src/configs/api.rs

```diff
@@ -224,6 +224,10 @@ pub struct Web3JsonRpcConfig {
     /// (hundreds or thousands RPS).
     #[serde(default)]
     pub extended_api_tracing: bool,
+    #[serde(default)]
+    pub deployment_allowlist_sink: bool,
+    pub http_file_url: Option<String>,
+    pub refresh_interval_secs: Option<u64>,
```

Comment:
These field names are too vague; e.g., nothing signals that `refresh_interval_secs` concerns the allowlist service specifically. I'd suggest to create a subconfig struct inside `Web3JsonRpcConfig` so that the allowlist params are well-structured:

```yaml
api:
  web3_json_rpc:
    deployment_allowlist:
      file_url: "http://example.com"
      refresh_interval_secs: 10
```